public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-new-site.php

```diff
@@ -615,6 +615,16 @@ protected function get_stub_pages( $wordcamp, $meta ) {
 					'wc_page_offline' => 'yes',
 				),
 			),
+
+			array(
+				'title'   => __( 'Day Of Event', 'wordcamporg' ),
+				'content' => $this->get_stub_content( 'page', 'day-of-event', $wordcamp ),
+				'status'  => 'publish',
```

Comment:
Should this be `draft` instead of `publish`, since the page is only intended to be used near the time of the actual event?

Reply:
oh, that's a good idea 👍